Cargo.toml

```diff
@@ -61,16 +61,16 @@ panic = "abort"
 codegen-units = 1
 
 [workspace.dependencies]
-ethers = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
-ethers-addressbook = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
-ethers-core = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
-ethers-contract = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
-ethers-contract-abigen = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
-ethers-providers = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
-ethers-signers = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
-ethers-middleware = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
-ethers-etherscan = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
-ethers-solc = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
+ethers = { version = "2.0.8", default-features = false }
+ethers-addressbook = { version = "2.0.8", default-features = false }
+ethers-core = { version = "2.0.8", default-features = false }
+ethers-contract = { version = "2.0.8", default-features = false }
+ethers-contract-abigen = { version = "2.0.8", default-features = false }
+ethers-providers = { version = "2.0.8", default-features = false }
+ethers-signers = { version = "2.0.8", default-features = false }
+ethers-middleware = { version = "2.0.8", default-features = false }
+ethers-etherscan = { version = "2.0.8", default-features = false }
+ethers-solc = { version = "2.0.8", default-features = false }
```

Comment:
now this is on ethers master, we should continue using git deps

Reply:
fixed in https://github.com/foundry-rs/foundry/pull/5397/commits/d92bd235fa551d073b7bfb09a3011a7d264387f5 — we can switch to master deps whenever we plan on publishing crates 😄